src/ring_theory/witt_vector/basic.lean

```diff
@@ -214,6 +214,7 @@ include hp
 
 local attribute [instance]
 private def comm_ring_aux₁ : comm_ring (𝕎 (mv_polynomial R ℚ)) :=
+by letI : comm_ring (mv_polynomial R ℚ) := mv_polynomial.comm_ring; exact
```

Comment:
Why is this necessary? Can that be understood?

Reply:
I'm not sure why this specific thing breaks, but this is the usual pi instance unification problem.